source/testnet/desktop-wallet/multisig-simple-transfer.rst

```diff
@@ -0,0 +1,131 @@
+=====================================================================
+
+.. contents::
+   :local:
+   :backlinks: none
+   :depth: 1
+
+Multi-signature GTU transfer
+============================
+
+A multi-signature transfer of GTU is a transaction that
+requires two or more co-signers to sign with their private keys to
+authorize the transaction.
+
+Prerequisites
+=============
+
+-  Set up an :ref:`identity and an account <create-initial-account-desktop>`.
+
+-  Set up a :ref:`Ledger Nano S hardware device identity <install-Ledger-app>`.
+
+Make a transfer proposal
+========================
+
+#.  Go to **Multi Signature Transactions**, and then select **Make new proposal**. All available transactions are listed in the right pane.
+
+#.  Select **Send GTU**.
+
+#.  Select the relevant proposer. You’ll see a list of all the identities that you’re the custodian of. When you select an identity, the **Transactions Details** in the left pane are updated with the identity information and the current transaction expiry time. You'll be able to change the expiry time later in this process. Use the scrollbar if you can't see all the information. Select **Continue**.
+
+#. Select the relevant sender account from the list of available accounts, and then select **Continue**.
+
+#.  In **Amount**, enter the amount of GTU you want to transfer. If the amount you want to transfer including the transfer fee exceeds the amount in your account, you’ll see a message saying **Insufficient funds**. This means that there's not enough GTU in the account and you’ll have to change the amount. Also, the amount must be greater than zero. In the **Transaction details** under **Amount**, you can see the estimated fee. Select **Continue**.
+
+#.  Select the relevant recipient from the list. If there are many recipients in the list, you can use search to find the right recipient. Select **Continue**.
+
+#. Set the **Transaction expiry** time and then select **Continue**.
+
+.. Note::
+   You must submit proposals to the chain within the last 2 hours up to the expiry date, so take this into consideration, when you set the expiry time. It's important that you leave enough time for the co-signers to return their signatures in time.
+
+Generate the transaction
+========================
+
+There are two ways that you can generate the transaction:
+
+-  :ref:`Generate the transaction proposal without signing<generate-transfer-without-sign>`. This option enables you to export the transaction proposal without signing it. You don't need a Ledger, but you do need an internet connection.
+
+-  :ref:`Generate and sign the transaction<generate-sign-transfer>` This option requires a Ledger but no internet connection.
+
+In combination, these two options enable organizations to distribute the responsibility of creating and signing transfers among more people. It makes it possible to have one employee create the proposals and another one sign the proposals.
```

Comment:
```suggestion
In combination, these two options enable organizations to distribute the responsibility of creating and signing transfers among more people. It makes it possible to have one employee create the proposals and another one sign the proposals.

Generating the proposal without immediately signing it with the Ledger, also enables organizations or other parties to generate proposals, and then take the proposals to sign at the location where the Ledger is stored. This could e.g. be in a bank vault, where an internet connection is not necessarily available.
```